corehq/motech/dhis2/migrations/0005_delete_jsonapilog.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.14 on 2018-07-13 11:26
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dhis2', '0004_auto_20171122_0827'),
+        ('motech', '0001_initial'),
+    ]
+
+    state_operations = [
+        migrations.DeleteModel(
+            name='JsonApiLog',
+        ),
+    ]
+    operations = [
+        migrations.SeparateDatabaseAndState(
```

Comment:
```RenameModel``` might have worked initially, but this makes sense now.

https://docs.djangoproject.com/en/2.0/ref/migration-operations/#renamemodel